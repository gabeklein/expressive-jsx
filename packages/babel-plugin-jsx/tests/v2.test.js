transform("whatever", `
  const App = () => {
    margin: 20;

    <div>Hello World</div>
  }
`)