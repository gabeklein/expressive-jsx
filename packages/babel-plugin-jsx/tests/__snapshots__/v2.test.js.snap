// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`whatever 1`] = `
  const App = () => {
    return <div className="App_8dv">Hello World</div>;
  };
  
  css(\`
  	.App_8dv {
  		margin: 20px;
  	}
  \`);
`;
